gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions-snapshots/gradle-5.0-20180923235913+0000-bin.zip
```

Comment:
Should we leave this on 4.10.2 for now so we don't have to release from a snapshot?